### DISCUSSION TITLE
为什么编译的时候说libncmdump.dll没有配置可执行文件无法启动

### DISCUSSION THREAD

**Discussion by HTstu** [taurusxin/ncmdump#22] [Q&A]
![image](https://github.com/user-attachments/assets/b54e5e0f-737b-46f1-afce-9edb56b414bf)
如题，而且releases里下载的ncmdump.exe似乎有相同的错误，运行直接闪退(×﹏×)

**Comment by taurusxin** [comment]
首先这是个命令行程序，你要从命令行运行，然后dll没法运行，是供其它应用调用的。
